src/store/servers.ts

```diff
@@ -13,28 +13,37 @@ export const servers: IServer[] = [
   {
     name     : 'Piplup (Main)',
     link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplup.smoo.it', ip: '143.198.241.242' },
-    location : { flag: 'gb', name: 'United Kingdom' },
+    server   : { host: 'piplup.smoo.it' },
+    location : { flag: 'fi', name: 'Finland' },
     version  : linkTree('server', 'piplup/server', 'TeamPiplup/SmoOnlineServer'),
     settings : {
       Server        : { MaxPlayers: 10 },
-      Scenario      : { MergeEnabled: false },
+      Scenario      : { MergeEnabled: true },
       Shines        : { Enabled: true },
       PersistShines : { Enabled: true },
     },
   },
   {
     name     : 'Piplup (Capture Sync)',
     link     : 'https://piplup.smoo.it',
-    server   : { host: 'piplup.smoo.it', ip: '143.198.241.242', port: 1028 },
-    location : { flag: 'gb', name: 'United Kingdom' },
+    server   : { host: 'piplup.smoo.it', port: 1028 },
+    location : { flag: 'fi', name: 'Finland' },
     version  : linkTree('server', 'piplup/server', 'TeamPiplup/SmoOnlineServer'),
+  },
+  {
+    name     : 'Piplup (UDP)',
+    link     : 'https://piplup.smoo.it',
+    server   : { host: 'us.smoo.raychu.xyz', port: 1029 },
```

Comment:
How about `piplUS.smoo.it` and/or `us.piplup.smoo.it` pointing to `us.smoo.raychu.xyz`?

Reply:
i mean yeah you can point a domain to `us.smoo.raychu.xyz` if you like